web/styles/subscriptions.css

```diff
@@ -1170,6 +1170,10 @@ div.settings-radio-input-parent {
             display: block;
             margin: 0 0 10px;
         }
+
+        .hide {
+            display: none;
+        }
```

Comment:
This can be moved inside the `.save-button-controls` CSS using `&.hide` selector, but this also seems fine.

Reply:
@sahil839 I have made the changes you suggested. Thanks to your suggestions, it is now more readable and organized.
The changes are ready for review now.